scripts/roll_release.sh

```diff
@@ -49,21 +56,14 @@ fi
 
 if ! [[ "$CURRENT_BRANCH" == "$RELEASE_BRANCH" ]]; then
     echo "Not on release branch (expected \"$RELEASE_BRANCH\", currently on \"${CURRENT_BRANCH}\"). Not continuing to make release."
-    exit 1
+    sanity_errors=$((sanity_errors + 1))
```

Comment:
I think the printout above needs to be amended, since the script is now allowed to continue past this point.

Reply:
improved output in general in https://github.com/eth-cscs/DLA-Future/pull/1296/commits/cd895576557466f31bd8d4939008cc687d03f1ee

Reply:
Cool!